tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/taxi_pipeline_regression_e2e_test.py

```diff
@@ -54,6 +55,24 @@ def setUp(self):
                                        self._pipeline_name, 'metadata.db')
     self._output_dir = os.path.join(self._test_dir, 'testdata')
 
+  def assertDirectoryEqual(self, dir1: Text, dir2: Text):
+    """Recursively comparing contents of two directories."""
+
+    dir_cmp = filecmp.dircmp(dir1, dir2)
+    self.assertEqual(len(dir_cmp.left_only), 0)
```

Comment:
How about using `assertEmpty` if possible? Same for below.

https://www.tensorflow.org/api_docs/python/tf/test/TestCase#assertEmpty